docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: TestCafe v1.13.0 Released
+permalink: /blog/:title.html
+---
+# TestCafe v1.13.0 Released
+
+This release adds support for custom paths to the configuration file, support for Microsoft Edge on Linux systems, and multiple bugfixes.
+
+<!--more-->
+
+## Enhancements
+
+### ⚙ Specify Custom Path to the TestCafe Configuration File ([PR #6035](https://github.com/DevExpress/testcafe/pull/6035) by [@Chris-Greaves](https://github.com/Chris-Greaves))
+
+TestCafe now enables you to specify a custom path to the [configuration file](../documentation/reference/configuration-file.md).
+
+To set the path, use one of the following options:
+
+* the [*--config-file* CLI option](../documentation/reference/command-line-interface.md#--config-file-path)
+* the [*configFile* option of the *createTestCafe* function](../documentation/reference/testcafe-api/global/createtestcafe.md#options)
+
+### Add Support for Microsoft Edge on Linux ([PR testcafe-browser-tools/#210](https://github.com/DevExpress/testcafe-browser-tools/pull/210) by [@dcsaszar](https://github.com/dcsaszar))
+
+If you follow the [Microsoft Edge Insider Channels for Linux](https://www.microsoftedgeinsider.com/en-us/download?platform=linux-deb) and have Microsoft Edge installed on your Linux machine, you can now launch TestCafe tests in this browser.
```

Comment:
Hm, looks like you should ***follow*** a channel to launch TestCafe tests...

Would it be clearer to say that you can launch tests in a browser version from the Insider channel (if that's what you mean)?

Reply:
> Would it be clearer to say that you can launch tests in a browser version from the Insider channel?

Yes, that's right.

Is that better:

```suggestion
If you have the [Insider version of the Microsoft Edge](https://www.microsoftedgeinsider.com/en-us/download?platform=linux-deb) installed on your Linux machine, you can now launch TestCafe tests in this browser.
```